src/mokka/src/test/java/pl/hycom/mokka/stubbing/transformer/GroovyResponseTransformerTest.java

```diff
@@ -0,0 +1,82 @@
+package pl.hycom.mokka.stubbing.transformer;
+
+import com.github.tomakehurst.wiremock.http.HttpHeaders;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.http.ResponseDefinition;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Bartosz Kuron (bartosz.kuron@hycom.pl)
+ */
+
+public class GroovyResponseTransformerTest {
+
+    private static final String URL = "/body";
+    private static final RequestMethod REQUEST_METHOD = RequestMethod.GET;
+    private static final String FROM = "000.00.0.000";
+    private static final String REQUEST_BODY_CONTENT = "request body content";
+
+    ResponseDefinition responseDefinition;
+
+    com.github.tomakehurst.wiremock.http.Request request;
+
+    private GroovyResponseTransformer groovyResponseTransformer = new GroovyResponseTransformer();
+
+    @BeforeEach
+    public void initialize(){
+        responseDefinition = mock(ResponseDefinition.class);
```

Comment:
Minor but could you please move this next to related `when` definitions? In general field should initialized as late as possible.